File: integrations/runtime-bridges/java/Bridge.java
package runtimebridges;

import java.util.Map;

public class Bridge {
    private Map<String, Object> config;

    public void init(Map<String, Object> config) {
        this.config = config;
    }

    public Map<String, Object> invoke(Map<String, Object> action) {
        return Map.of("status", "todo", "action", action);
    }

    public Map<String, Object> stream() {
        return Map.of("status", "todo", "event", "stream-not-implemented");
    }

    public void close() {
        // no-op
    }
}

File: integrations/runtime-bridges/dotnet/Bridge.cs
using System.Collections.Generic;

namespace RuntimeBridges;

public class Bridge
{
    private Dictionary<string, object> _config = new();

    public void Init(Dictionary<string, object> config) => _config = config;

    public Dictionary<string, object> Invoke(Dictionary<string, object> action) => new()
    {
        ["status"] = "todo",
        ["action"] = action
    };

    public Dictionary<string, object> Stream() => new()
    {
        ["status"] = "todo",
        ["event"] = "stream-not-implemented"
    };

    public void Close() { }
}
